scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -83,6 +92,71 @@ def _build_arg_parser():
     return p
 
 
+def binarize_labels(atlas, label_list):
+    #Input atlas must be a numpy array
```

Comment:
PEP 8: block comment should start with '#' (i.e. add a whitespace between the hashtag and the text)

Reply:
Done in dbd0b2a.